services/worker-manager/test/fakes/azure.js

```diff
@@ -232,13 +233,21 @@ class VMResourceManager extends ResourceManager {
   // Subclass Overrides
 
   _requestToResource(request) {
+    let dataDisks = [];
+    for (let i = 0; i < request.parameters.storageProfile.dataDisks.length; i++) {
+      dataDisks.push({name: slugid.nice()});
```

Comment:
I kind of want these disks to also appear in the disk manager, but I see that for the tests as they are this doesn't matter (because we test provisioning and removing separately).  Maybe a comment to help someone later wondering why they don't exist?